watch-package.js

```diff
@@ -57,8 +59,17 @@ module.exports = function watchPackage (pkgDir, exit) {
       return a.concat(b)
     })
 
+    if (ignores) {
+      ignores = [].concat(ignores).map(function (ignore) {
+        return ['--ignore', ignore]
```

Comment:
I don't think they need quoting. That's so they're protected from shell expansion, but we're not executing nodemon through the shell, so the argument will go through with wildcards intact. Or am I wrong?